.rubocop.yml

```diff
@@ -39,3 +39,8 @@ AllCops:
     - db/schema.rb
     - db/migrate/*
     - vendor/**/*
+
+# Let eval work its magic in the specs
+Lint/Eval:
+  Exclude:
+    - 'spec/**/*'
```

Comment:
Can you give me a hint on that inline comment? Then I'll just revert delete the offending commit and use that.